radio/src/CMakeLists.txt

```diff
@@ -393,7 +393,7 @@ endforeach()
 add_definitions(-DCORRECT_NEGATIVE_SHIFTS)
 
 if(NOT MSVC)
-  set(WARNING_FLAGS "${WARNING_FLAGS} -Wall -Wno-strict-aliasing -Wformat -Wreturn-type -Wunused -Wuninitialized -Wunknown-pragmas -Wno-switch -Wtype-limits")
+  set(WARNING_FLAGS "${WARNING_FLAGS} -Wall -Wno-strict-aliasing -Wformat -Wreturn-type -Wunused -Wuninitialized -Wunknown-pragmas -Wno-switch -Wtype-limits -Wno-attributes")
```

Comment:
What does disabling this warning do?

Reply:
some compilers will raise a warning when seeing `[[fallthrough]];` (which is called an attribute)

Reply:
So we're silencing a potentially useful compiler warning so that the code scanner doesn't issue a bogus warning?

Reply:
no, this is gcc which issues a bogus warning, not sonar!

Reply:
I understand, I just meant that `[[fallthrough]]` was to silence a Sonar warning and then this switch is needed to silence warnings about `[[fallthrough]]`.  :)